scripts/generate-docfx-yml.ps1

```diff
@@ -0,0 +1,61 @@
+
+# Usage: ./generate-docfx-yml.ps1 <directoryPath>
+#
+# Recursively iterates directoryPath and creates a toc.yml in each subdirectory,
+# generating a table of contents for each markdown file. This is useful for docfx
+# as docfx requires a manifest for just in time static site compilation.
+#
+
+param (
+    [string]$directoryPath
```

Comment:
Any reason why this shouldn't be hardcoded or at least defaulted to the `docs` folder?

Reply:
Oh I wanted to keep it as generic as possible so other folks interested in docfx can reuse it and follow our workflow of dynamically generating toc.yml if they don't want to manually maintain their own yamls.